src/components/Main/Main.js

```diff
@@ -1,9 +1,128 @@
 import React from 'react';
 
+import { Button } from '@material-ui/core';
+import { lessonSlides } from '../../constants/lessons.js';
+import LessonText from '../LessonText/LessonText.js';
+import Grid from '@material-ui/core/Grid';
+
 class Main extends React.Component {
-    render = () => {
-        return(<div>owo</div>);
-    }
+	constructor(props) {
+		super(props);
+		this.state = {
+			count: 0,
+			lessonNum: 0,
+      value: '',
+			userInput: '',
+			inputError: false,
+			errorString: ''
+		};
+		this.handleInputChange = this.handleInputChange.bind(this);
+		this.handleInputSubmit = this.handleInputSubmit.bind(this);
+		this.setCount = this.setCount.bind(this);
+		this.setLessonNum = this.setLessonNum.bind(this);
```

Comment:
I would probably just use arrow functions instead of manual binding? The ES6 (7?) transpiler automatically converts it anyways, and it's harder to forget.

Reply:
Ok, sounds good!